clang/test/Driver/darwin-ld-platform-version-macos.c

```diff
@@ -43,8 +43,8 @@
 
 // RUN: %clang -target x86_64-apple-macos10.13 -mlinker-version=520 \
 // RUN:   -### %t.o 2>&1 \
-// RUN:   | FileCheck --check-prefix=NOSDK %s
+// RUN:   | FileCheck --check-prefix=INFERRED-SDK %s
```

Comment:
I think you want to delete those two. The test will fail if no CLANG_USE_XCSELECT is used.

Reply:
I think they'll pass since I've just relaxed the text to match against. But I'll double check locally for both enabling and disabling `CLANG_USE_XCSELECT` and delete them if it fails.

Reply:
I see. The value of the test is to check the behavior of matching the version when SDK is not provided. I don't think current test provide any value.

Reply:
I think it does provide some value, particularly for builds with `CLANG_USE_XCSELECT=ON` (since there is a stricter test that was separated out for builds where it is disabled).

That is, we check that the first two arguments to `-platform_version`, "macos" and "10.13.0", are correctly inferred from `-target`, and that the last argument looks like a version scheme, even when no `-isysroot` flag is provided.

That still seems valuable to me, even if it's not the same as the original test.

However, if you're still not convinced, I can remove it.